sdk/python/test/e2e/utils.py

```diff
@@ -68,8 +75,7 @@ def verify_job_e2e(
 def get_pod_spec_scheduler_name(gang_scheduler_name: str) -> str:
     if gang_scheduler_name == TEST_GANG_SCHEDULER_NAME_SCHEDULER_PLUGINS:
         return DEFAULT_SCHEDULER_PLUGINS_NAME
-    # TODO (tenzen-y): Implement E2E tests using volcano.
     elif gang_scheduler_name == TEST_GANG_SCHEDULER_NAME_VOLCANO:
-        return ""
+        return TEST_GANG_SCHEDULER_NAME_VOLCANO
```

Comment:
@johnugeorge In fact, even though we forgot to set `volcano` to `schedulerName` in the podSpec, e2e passed in #1831.

Reply:
This is necessary and it used to set the scheduler for gang-schedule e2e. I forget to changed it in last pr, sorry.